DOC-2691-Differential backup [3.10] (#805)
tigergraph/server-docs: modules/backup-and-restore/pages/online-backup.adoc

== Limitations and Considerations

Currently, the Online Backup feature has one limitation:

> * If users start running a query, the full backup cannot succeed until that query finishes.

Reviewer: ```suggestion
* If a query starts during a *full* backup, the backup will not finish until that query finishes.
```

> However, the *differential backup* does not have this limitation. Users can perform differential backups even if a query is running.

Reviewer: ```suggestion
However, *differential backup* does not have this limitation. Users can perform differential backups even if a query is running.
```
